cookbook-react/src/recipes/inputs/selectable-list/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+
+import SelectableList, { SelectableChildBaseProps } from './components/SelectableList';
+import styles from './styles.module.scss';
+
+/**
+ * This component is an example. The real component is the SelectableList in the components folder
+ */
+
+interface ItemType {
+  id: string;
+  name: string;
+  label: string;
+}
+
+const items = [
+  { id: '1', name: 'first_item', label: 'First item' },
+  { id: '2', name: 'second_item', label: 'Second item' },
+  { id: '3', name: 'third_item', label: 'Third item' }
+];
+
+// If I don't do this, input names and ids collide since there are multiple lists in this example.
+const listPrefix = (listId: string, value: string) => `${listId}-${value}`;
```

Comment:
I'd just remove this and interpolate in place, I think it'd be simpler.

Reply:
I was originally doing that but I found this a little less error prone, idk.

Reply:
It's fine